cypress/e2e/accessibility.cy.ts

```diff
@@ -2,6 +2,7 @@ describe("Accessibility tests in light mode", () => {
     it("Checks clients page", () => {
         cy.login();
         cy.visit("/clients");
+        cy.get("h1", { timeout: 5000 }).should("exist");
```

Comment:
This waits for the page to be hydrated. I've added it in the tests that need it for now. Other tests may also need it in the future.